packages/components/src/components/wordmark/wordmark.tsx

```diff
@@ -0,0 +1,70 @@
+import type { PropTypes, SelectedAriaAttributes, Theme } from '../../types';
+import type { WordmarkSize, WordmarkTarget, WordmarkAriaAttribute } from './wordmark-utils';
+import { Component, Element, h, Host, JSX, Prop } from '@stencil/core';
+import { AllowedTypes, attachComponentCss, parseAndGetAriaAttributes, validateProps, THEMES } from '../../utils';
+import { WORDMARK_ARIA_ATTRIBUTES, WORDMARK_SIZES } from './wordmark-utils';
+import { getComponentCss } from './wordmark-styles';
+
+const propTypes: PropTypes<typeof Wordmark> = {
+  size: AllowedTypes.oneOf<WordmarkSize>(WORDMARK_SIZES),
+  theme: AllowedTypes.oneOf<Theme>(THEMES),
+  href: AllowedTypes.string,
+  target: AllowedTypes.string,
+  aria: AllowedTypes.aria<WordmarkAriaAttribute>(WORDMARK_ARIA_ATTRIBUTES),
+};
+
+@Component({
+  tag: 'p-wordmark',
+  shadow: { delegatesFocus: true },
+})
+export class Wordmark {
+  @Element() public host!: HTMLElement;
+
+  /** Adapts sizing of wordmark. */
+  @Prop() public size?: WordmarkSize = 'small';
+
+  /** Adapts color depending on theme. */
+  @Prop() public theme?: Theme = 'light';
+
+  /** When providing an url then the component will be rendered as `<a>`. */
+  @Prop() public href?: string;
+
+  /** Target attribute where the link should be opened. */
+  @Prop() public target?: WordmarkTarget = '_self';
+
+  /** Add ARIA attributes. */
+  @Prop() public aria?: SelectedAriaAttributes<WordmarkAriaAttribute>;
+
+  public render(): JSX.Element {
+    validateProps(this, propTypes);
+    attachComponentCss(this.host, getComponentCss, this.size, this.theme);
+
+    const svg = (
+      <svg viewBox="0 0 4383 300" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please add the raw svg and the optimized one as assets and comment how the optimized one was created with. Similar to stepper-horizontal or model-signature package.